src/undo_log/undo_log.c

```diff
@@ -318,6 +320,8 @@ void UndoLog_Rollback
 		}
  	}
 
+	QueryCtx_ForceUnlockCommit();
```

Comment:
// force unlock, as we're not executing within the context of an operation
// assumption: no operations should be executing at this point